app/components/media/description_component.html.erb

```diff
@@ -0,0 +1,3 @@
+    <p class="prose prose-gray dark:prose-invert text-gray-500 dark:text-gray-400 <%= classes %>">
+      <%= @media[:description] %>
+    </p>
```

Comment:
We need a final newline here to keep the linter happy. Causing CI to fail

Reply:
![Screenshot from 2023-07-19 22-41-19](https://github.com/TheOdinProject/theodinproject/assets/43343880/27f5dfab-c123-4061-941f-bb2ed8246de8)

Huh, not sure why `yarn eslint` wasn't showing any warnings locally, didn't think to look at the ci - my bad! Addressed this in https://github.com/TheOdinProject/theodinproject/pull/3926/commits/db8e1fe866fa7c008677db7ef08b1ca782842597